### PR TITLE
Koa/koa 4961 simplifying example project

### PR DESCRIPTION
**Pull request by frugoman**
Cleaning and simplifying the Example project quite a bit to make it easier to maintain and iterate

### CODE REVIEW

File: Example/Backpack/FeatureStories/TokenCells.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ */
+
+struct TokenCellsProvider {
+    let navigator: PresentableNavigator
+    
+    private func show(presentable: Presentable) {
+        navigator.present(presentable: presentable)
+    }
+    
+    private func showChilds(for title: String, childs: [Components.Group]) {
```
Comment (jookes) [inline Example/Backpack/FeatureStories/TokenCells.swift:26]: showChildren might be a nicer naming.

**Review by jookes — COMMENTED** [review]
Just a few tiny nitpicks on naming. Like the simpler way of setting up the stories!

File: Example/Backpack/AppDelegate.swift [hunk 2]
```diff
@@ -27,23 +26,8 @@ import Backpack
...
-        let relativeFontDefinition = BPKRelativeFontDefinition()
-        let relativeTestFont = UIFont(name: relativeFontDefinition.regularFontFace, size: 12)
-        let relativeAvailable = relativeTestFont != nil
-        let useRelative = !ProcessInfo.processInfo.arguments.contains("DISABLE_RELATIVE") && relativeAvailable
-        
-        if useRelative {
-            BPKFont.setFontDefinition(relativeFontDefinition)
-        }
-        
+    fileprivate func setupAppearance() {
```
[the remark this answers is not in the document]
Comment (frugoman) [reply Example/Backpack/AppDelegate.swift:30]: right, it's Xcode refactor tool that does this `fileprivate` thing automatically. Fixed!

**Review by jookes — APPROVED** [review]
Looks good to me!

**Pull request merged**